doc/source/devices/adi.ad7625.rst

```diff
@@ -0,0 +1,20 @@
++ad7625
++=================
++
++By default, the device_name parameter in the class constructor is the
++same as the class name. To use the class with another supported model,
++the name must be given when instantiating the object. For example, if
++working with an ad7626 with a URI of "10.2.5.222", use the ad7625 class,
```

Comment:
Change URI 10.2.5.222 to ip:10.2.5.222 or say "using the IP backend with address". End users tend to screw up URI definitions and I just want things to be as clear as possible